implementations/elixir/ockam/ockam_hub/config/runtime.exs

```diff
@@ -155,3 +155,27 @@ config :ockam_kafka,
     client_config: kafka_client_config,
     replication_factor: kafka_replication_factor
   ]
+
+## Services config
+
+## This may be default of on-demand nodes
+kafka_service_prefix = System.get_env("KAFKA_SERVICE_PREFIX") || ""
+
+config :ockam_hub,
```

Comment:
The idea here is that `service_providers` are defined per add-on artifact (e.g. ockam-node-kafka docker image)
while `services` is provided by the user (via hub-console) and declaratively defines the services (with arguments) to run on the node.

Reply:
How would be the service definition procedure from ockam hub node perspective?

Reply:
The options I though were environemnt variable or a file in some format (e.g. json). 
This may be accessed from `Ockam.Hub.Service.Provider`, does not have to be from `runtime.exs`.
A file makes it easier to deal with secrets since we can encrypt it.